cmd/operator/main.go

```diff
@@ -10,13 +10,20 @@ import (
 
 func main() {
 	configFilePath := flag.String("config", "config.json", "config file path")
+	logLevel := flag.String("log-level", "info", "log level")
 	flag.Parse()
 
 	cfg, err := operator.LoadConfig(*configFilePath)
 	operator.AssertNoError(err)
 	log.Println("Config loaded")
 
-	_operator, _ := operator.Setup(cfg)
+	lvl, err := log.ParseLevel(*logLevel)
+	if err != nil {
+		log.Fatalf("parsing log level: %w", err)
```

Comment:
```suggestion
		log.Fatalf("parsing log level: %v", err)
```
`%w` only works with `fmt.Errorf` error wrapping when passing errors up the stack.

Reply:
got it, done